src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1771,4 +1771,7 @@ Additional information: {1}</value>
   <data name="Visual_Studio_Settings" xml:space="preserve">
     <value>Visual Studio Settings</value>
   </data>
+  <data name="Show_hints_for_array_indexers" xml:space="preserve">
+    <value>Show hints for array indexers</value>
```

Comment:
Since types other that Arrays can have indexers, I wonder if `Show hints for indexer parameters` would be a better option name.

Reply:
Yeah that is better, I'll change that